store/src/test/java/com/nytimes/android/external/store3/ClearingPersister.java

```diff
@@ -0,0 +1,29 @@
+package com.nytimes.android.external.store3;
```

Comment:
Is that a new class? Seems that it has been created on the Kotlin conversions scope, but not in Kotlin! I think it can be Kotlin as well, or was there any other blocker @brianPlummer ?

Reply:
this was a static inner class declared in one of the java tests, there was a kotlin dep and it failed to resolve it, unwinding to java here and will convert to kotlin in next pass.  good eye!